src/terminal/char_grid/styles.rs

```diff
@@ -29,9 +29,25 @@ pub struct Styles {
     pub strikethrough: bool,
     pub inverted: bool,
     pub blink: bool,
+    config: Config,
```

Comment:
My intent was originally to just get this working then later (in this same PR) once the `Config` is being initialized in `scaffolding` to experiment with memory optimization by keeping references to a per-`Terminal` config.